server/src/main/java/com/kritsit/casetracker/server/datalayer/UserRepository.java

```diff
@@ -120,7 +121,7 @@ public Staff getInvestigatingOfficer(String caseNumber) throws RowToModelParseEx
                 inspectors.add(getUserDetails(inspector.get("username")));
             }
             return inspectors;
-        } catch (Exception e) {
+        } catch (SQLException e) {
```

Comment:
The other option is to follow the pattern of the rest of the file - catch RuntimeException and simply throw it again and then catch Exception and throw RowToModelParseException ([see here](https://github.com/paddatrapper/CaseTracker/pull/25#discussion-diff-32841340R55))